administrator/language/de-DE/plg_task_rotatelogs.sys.ini

```
@@ -0,0 +1,8 @@
+; Joomla! German Translation
+; (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
+; (C) Translation 2008 - 2023 J!German <https://www.jgerman.de>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_TASK_ROTATELOGS="Aufgabe – Protokollrotation"
+PLG_TASK_ROTATELOGS_XML_DESCRIPTION="Bietet Aufgabenroutinen zum periodischen Rotieren der Systemprotokolldateien."
```

```suggestion
PLG_TASK_ROTATELOGS_XML_DESCRIPTION="Bietet Aufgabenroutinen zum periodischen Rotieren der Protokolldateien."
```
sync mit plg_task_rotatelogs.ini